contracts/protocols/mainnet/liquity.sol

```diff
@@ -120,6 +166,30 @@ contract Resolver is Helpers {
     bool isInRecoveryMode = troveManager.checkRecoveryMode(oracleEthPrice);
     return System(borrowFee, ethTvl, tcr, isInRecoveryMode);
   }
+
+  function getTrovePositionHints(uint collateral, uint debt, uint searchIterations) external view returns (
+    address upperHint,
+    address lowerHint
+  ) {
+    // See: https://github.com/liquity/dev#supplying-hints-to-trove-operations
+    uint nominalCr = hintHelpers.computeNominalCR(collateral, debt);
+    searchIterations = searchIterations == 0 ? mul(10, sqrt(sortedTroves.getSize())) : searchIterations;
+    (address hintAddress, ,) = hintHelpers.getApproxHint(nominalCr, searchIterations, 3);
```

Comment:
Why `3`? Maybe it would be nice to put it as a constant defined at the beginning.

Reply:
Maybe it should default to something pseudo-random (latest block hash or something), but be overrideable (for the sake of predictable tests).

Otherwise it will always result in the same sequence of picks as long as the total number of Troves remains the same.

Reply:
Would that matter? This function (unlike `getApproxHint`) isn't resumable. I guess it could get unlucky and get stuck finding a bad hint until the system grew.

Reply:
Thanks, I've added randomSeed as an optional parameter: https://github.com/liquity/dsa-resolvers/pull/2/commits/b471eab625fbdd54fde605df22ccd824918129ba